packages/conventional-changelog-core/test/test.js

```diff
@@ -269,7 +269,7 @@ describe('conventionalChangelogCore', function() {
       .pipe(through(function(chunk) {
         chunk = chunk.toString();
 
-        expect(chunk).to.include('## 0.0.17');
+        expect(chunk).to.include('## <small>0.0.17');
```

Comment:
Is there anyway we can check that `</`small>` is also in the title line?

(Actually, it looks like you test this below for `conventional-changelog-writer`. So would it benefit us to verify the same behavior in `core`?)